src/io_threads.c

```diff
@@ -297,6 +303,40 @@ void killIOThreads(void) {
     }
 }
 
+int updateIOThreads(const char **err) {
+    serverAssert(inMainThread());
+    UNUSED(err);
+    int prev_threads_num = 1;
+    for (int i = IO_THREADS_MAX_NUM - 1; i > 0; i--) {
+        if (io_threads[i]) {
+            prev_threads_num = i + 1;
+            break;
+        }
+    }
+    if (prev_threads_num == server.io_threads_num) goto done;
```

Comment:
```suggestion
    if (prev_threads_num == server.io_threads_num) return 1;
```
There is no other logic in done, so we can just return here.